doc/source/whatsnew/v0.25.1.rst

```diff
@@ -105,6 +105,7 @@ I/O
 
 - Avoid calling ``S3File.s3`` when reading parquet, as this was removed in s3fs version 0.3.0 (:issue:`27756`)
 - Better error message when a negative header is passed in :func:`pandas.read_csv` (:issue:`27779`)
+- Added notes on Engine Disposal and user responsibility to multiple sql related function docstrings (:issue:`23086`)
```

Comment:
@TomAugspurger : Do we document entries for this?

Reply:
Yes.